lib/src/package_analyzer.dart

```diff
@@ -142,11 +142,10 @@ class PackageAnalyzer {
     if (pkgResolution != null) {
       List<CodeProblem> analyzerItems;
       if (dartFiles.isNotEmpty) {
-        analyzerItems = await context.staticAnalysis();
-      } else {
-        analyzerItems = <CodeProblem>[];
```

Comment:
I think this part may be important, but may not be anymore....

Reply:
We have a null-check just below, and don't use the variable after that. I think it is all right.